ansible/linux-webconsole.yml

```diff
@@ -3,4 +3,5 @@
 - hosts: all
   become: yes
   roles:
+    - linux-common
```

Comment:
This is looking good to me, but clearly we need a rebase and we need to include the K8s builds.
After the rebase I think the K8s builds are already using linux-common, so that might just work 🤞